app/loaders/browser-loader.js

```diff
@@ -88,7 +89,7 @@ class BrowserLoader {
 		});
 
 		let isMac = process.platform === "darwin";
-		shortcuts.register(win, isMac ? "Command+Option+I" : "Control+Shift+I", () => contents.toggleDevTools());
+		shortcuts.register(win, [isMac ? "Command+Option+I" : "Control+Shift+I", "F12"], () => WindowUtils.openDevtools(win));
```

Comment:
This is a breaking change. We want this keybind to toggle the DevTools, not to open another one.